iOS/docs/kickoff/configure-ci/README.md

```diff
@@ -0,0 +1,171 @@
+
+## Check configuration files encryption...
+
+This step is optional and only useful if you need to encrypt files to be used in Bitrise. As long as you use CI only for compiling and running tests, this step will not be necessary. Instead, if you configure CI for building the application (for example for distribution) you will need to upload the encrypted `.xcconfig` files.
+
+The provided `bitrise.yml` is not configured for encryption, so in case your project requires it follow these steps:
+
+### Encryption
+
+If you need to use any secret configuration keys in the Bitrise build, encrypt the `.xcconfig` files with those keys.
+
+```
+# Put all the files in a tar file
+tar cvf secrets.tar file1 file2 
+
+# Encrypt the tar file
+openssl aes-256-cbc -k "password" -in secrets.tar -out secrets.tar.enc 
+```
+
+### Decryption
+
+Add this as first step of the `bitrise.yml`:
+
+```
+    - script@1.1.3:
+        title: Decrypting files...
+        inputs:
+        - content: openssl aes-256-cbc -k "$ENCRYPT_PASSWORD" -in secrets.tar.enc -out secrets.tar -d && tar xvf secrets.tar
+```
+
+Don’t forget to include `ENCRYPT_PASSWORD` as explained in [Configure App Environment variables](#configure-app-environment-variables)
+
+## Create Bitrise app...
+
+### Log in to [Bitrise](https://www.bitrise.io/) using your Wolox's github account.
+
+If you’re not added to the `Wolox` organization, you should ask the technical leader to do so.
+
+### Add application
+
+In Bitrise dashboard, press "Add new app" to start the process. 
+
+### Choose project’s repository
+
+Before starting, make sure `Wolox` is selected in the "Add new app to" option in top left.
+
+![choose organization](./resources/br1.png)
+
+From the list of platforms, choose `Github`. Below, in the left part of the screen you can see a list of github organizations you contribute to. Choose `Wolox` (or the organization the repository is under), and then at the right choose the repository you want to integrate.
+
+![choose project](./resources/br2.png)
+
+### Setup repository access
+
+Select `auto-add SSH Key`. The key should be added automatically after a moment. If it doesn’t work, you may need to do it manually.
+
+NOTE: Only Github admins can automatically add the SSH key in this step. In case you have no access, skip this step and ask your TL to do it.
+
+![choose project](./resources/br3.png)
+
+### Validation setup
+
+Choose the main branch of your project (master or development usually). Bitrise will scan the branch to check for the correct configurations. This takes about 5 minutes.
+
+### Project build configuration
+
+Select iOS. Verify that the correct XCode stack is selected and confirm.
+
+![choose project](./resources/br4.png)
+
+### Webhook setup
+
```

Comment:
I'd add something like: 'Please note this can only be done by a repository administrator.' to remove the duplication mention about this in both paragraphs.